### PR TITLE
fix(csp): use lottie-light to avoid eval()

### PR DESCRIPTION
**Pull request by robstax**
# IMPORTANT

**Currently, this project is closed to any external contributions. Any pull request made against this project from external sources will likely be closed. If you would like to make changes to this project, please fork this project.**

# Guide

**This "Help" section can be deleted before submitting this pull request.**

*Update the name of this pull request to reflect the following shape:*

```
{type}/{scope?}/{message}
```

* **type** - A [conventional commit type](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional#type-enum) **REQUIRED**
* **scope** - The kabab-case scope of the changes in this request
* **message** - A short, kebab-case statement describing the changes **REQUIRED**

*Provide a general summary of the scope of the changes in this pull request.*

* [Readme](https://github.com/momentum-design/momentum-react-v2/blob/master/README.md)
* [Getting Started Guide](https://github.com/momentum-design/momentum-react-v2/blob/master/GETTING_STARTED.md)
* [Contributing Guide](https://github.com/momentum-design/momentum-react-v2/blob/master/CONTRIBUTING.md)

# Description
lottie uses `eval()` which is not considered secure and will cause issues with CSP. it uses this to evaluate expressions on animations. i don't believe we use any of that functionality and lottie-light does not use eval()

# Links
https://github.com/airbnb/lottie-web/issues/2173

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true,
+    "source.fixAll.eslint": "explicit"
```
Comment (torgeadelin) [inline .vscode/settings.json:3]: this has been haunting me jesus

File: src/components/Reaction/Reaction.tsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React, { FC, useRef, useEffect } from 'react';
 import classnames from 'classnames';
-import lottie, { AnimationItem } from 'lottie-web';
+import lottie, { AnimationItem } from 'lottie-web/build/player/lottie_light';
```
Comment (torgeadelin) [inline src/components/Reaction/Reaction.tsx:3]: not a fan of importing things like this, is it not distributed in any other way?

**Review by torgeadelin — COMMENTED** [review]
lgtm, just some small comment on the import. will need to test locally and then can approve

File: src/components/Reaction/Reaction.tsx (hunk 2 above)
Comment (robstax) [reply src/components/Reaction/Reaction.tsx:3]: i agree. i could not find any documentation on how else to do this. most GH issues were doing it this way or through webpack like `'lottie-web': path.resolve('node_modules/lottie-web/build/player/lottie_light.js'),`

Comment (torgeadelin) [reply src/components/Reaction/Reaction.tsx:3]: if there's no official way it's fine..

Comment (InteractiveTimmy) [reply src/components/Reaction/Reaction.tsx:3]: After inspecting the package definition, it appears that this is the best approach to using `lottie_light`, as it is not defined in the entry point or from the package.json.

**Review by torgeadelin — APPROVED** [review]

**Pull request merged**